closure/protobuf/closure_proto_library.bzl

```diff
@@ -93,6 +93,10 @@ def _closure_proto_aspect_impl(target, ctx):
   srcs = depset([js])
   deps = unfurl(ctx.rule.attr.deps, provider="closure_js_library")
   deps += [ctx.attr._closure_library, ctx.attr._closure_protobuf_jspb]
+  proto_deps = unfurl(ctx.rule.attr.deps, provider="proto")
+  descriptors = depset(
+    direct = target.files.to_list(),
+    transitive=[d.proto.transitive_descriptor_sets for d in proto_deps])
```

Comment:
I think it should be sufficient to skip the `unfurl` and use
```
descriptors = depset(
  direct = target.proto.direct_descriptor_set,
  transitive = target.proto.transitive_descriptor_sets)
```

Reply:
Thanks!  Added a commit to address this (both arguments want a sequence though).